webapp/security/views.py

```diff
@@ -190,20 +190,9 @@ def api_create_notice():
     if not flask.request.json:
```

Comment:
^ could we just call this `create_notice`? I'm not sure what the benefit is of conceptually splitting "api functions" from other functions. If we were going to do that, you'd think we might as well be keeping this at `api.ubuntu.com` or `ubuntu.com/api`, which we're deliberately not.

To me, the whole thing is a single web service, which accepts GETs, POSTs, PUTs etc., and talks HTML and JSON where appropriate.

Reply:
updated